webpack.common.config.js

```diff
@@ -14,6 +14,11 @@ module.exports = {
       {
         test: [/\.less$/, /\.css$/],
         loader: "style-loader!css-loader!less-loader"
-      }]
+      },
+      {
+        test: /\.pegjs$/,
```

Comment:
I thought that this webpack config (instead of using the loader inline) was a little easier to manage. I can change it to the suggested inline (`require('pegjs-loader!./grammar.pegjs')` if you'd like though